### ISSUE TITLE
Is this process still valid after your 2018 post?

### ISSUE THREAD

**Issue by paulobriennz** [joepitt91/SignClickOnceApp#3]
Hi Joe, first off.  Thank you, you seem to be the only human on the planet who has documented how to sign a click once app with a REAL cert, not a homemade one.

But, for the life of me, i cannot get it to work.. No errors, all says its worked, EXEs are signed and look good, but still come up with unknown publisher.

I found your article asking 'what is the next hoop to jump through'... did you ever get a response?
I'm trying to figure out if this script is 'post' that article (and solves whatever the issue is) or pre, and doesn't take in to consideration what Windows has changed now...

https://social.msdn.microsoft.com/Forums/windows/en-US/43cafbe1-1ee4-452f-8e74-d9d4dab7e1d5/smartscreen-quotunknown-publisherquot-with-signed-clickonce-exe-amp-manifest?forum=winformssetup

Any help appreciated - tearing my hair out.  It all looks like its worked, but it hasn't.  I'm using a purchased EV code signing cert from Sectigo.

**Comment by joepitt91** [comment]
It has been a while since I've needed to do this, but I seem to remember that the latest version of VS will sign them properly with a SHA256 cert out of the box. the issue now is that even valid certificates do not get you an automatic pass through SmartScreen, I'll do some testing when I get a chance and update here.

**Comment by joepitt91** [comment]
If you are using VS2019, are you timestamping as well as signing, using a SHA256 certificate and timestamping does not present any certificate warnings for me.

Test setup:

![image](https://user-images.githubusercontent.com/16159859/80316058-7ea88f00-87f3-11ea-8751-179c4533e9b7.png)

**Comment by paulobriennz** [comment]
Thanks for replying Joe.  Yes I sorted it in the end. Yes, it seems VS2019 will do without any other process, but I think my issue was not having the windows 10 sdk installed. Once I installed, and specified time server, things worked ok. I think I was initially just running the published setup.exe from local pc and getting unknown publisher, but once I deployed to a web server and tested from there, it was all good.
